components/google_calendar/README.md

```diff
@@ -18,3 +18,5 @@ include:
   to see their tasks and events in one place.
 - A reminder application that uses Google Calendar data to remind users of
   upcoming events.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Reply:
Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).